drakrun/drakrun/playground.py

```diff
@@ -16,13 +19,27 @@
 from drakrun.draksetup import find_default_interface
 
 
+def cleanup():
+
+    logging.info("Ensuring that drakrun@* services are stopped...")
+    try:
+        subprocess.check_output('systemctl stop \'drakrun@*\'', shell=True, stderr=subprocess.STDOUT)
+    except subprocess.CalledProcessError:
+        logging.error("drakrun services not stopped")
+        return
```

Comment:
Why stop all instances? We're using only single instance

Reply:
Makes sense, will update it to stop the instance we want to run